src/components/SignUpForm/index.jsx

```diff
@@ -1,5 +1,150 @@
+  });
+
+  const onSubmit = (data) => {
+    submit(data);
+    reset();
+  };
+
+  return (
+    <div className="signUpForm">
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div className="formInputGroup">
+          <div className="formInput">
+            <input
+              className={errors.email && "inputError"}
+              type="text"
+              placeholder="First name"
+              name="firstName"
+              aria-label="firstName"
+              ref={register()}
+            />
+            {errors?.firstName && (
+              <InputErrorMessage message={errors.firstName.message} />
+            )}
+          </div>
+
+          <div className="formInput">
+            <input
+              className={errors.email && "inputError"}
+              type="text"
+              placeholder="Last name"
+              name="lastName"
+              aria-label="lastName"
+              ref={register()}
+            />
+            {errors?.lastName && (
+              <InputErrorMessage message={errors.lastName.message} />
+            )}
+          </div>
+        </div>
+
+        <div className="formInput">
+          <input
+            className={errors.email && "inputError"}
+            type="email"
+            placeholder="Your email"
+            name="email"
+            aria-label="email"
+            ref={register()}
+          />
+          {errors?.email && (
+            <InputErrorMessage message={errors.email.message} />
+          )}
+        </div>
+
+        <div className="formInputGroup">
+          <div className="formInput">
+            <input
+              className={errors.password && "inputError"}
+              type="password"
+              placeholder="Password"
+              name="password"
+              aria-label="password"
+              ref={register()}
+            />
+            {errors?.password && (
+              <InputErrorMessage message={errors.password.message} />
+            )}
+          </div>
+
+          <div className="formInput">
+            <input
+              className={errors.confirmPassword && "inputError"}
+              type="password"
+              placeholder="Repeat password"
+              name="confirmPassword"
+              aria-label="confirmPassword"
+              ref={register()}
+            />
+            {errors?.confirmPassword && (
+              <InputErrorMessage message={errors.confirmPassword.message} />
+            )}
+          </div>
+        </div>
+
+        <div className="formInput">
+          <label htmlFor="acceptTerms">
+            <input
+              className={errors.acceptTerms && "inputError"}
+              type="checkbox"
+              id="acceptTerms"
+              name="acceptTerms"
+              value="accepted"
+              ref={register()}
+            />
+            I have read and agree with the&nbsp;
+            <a href="/terms">Terms of Service</a>
+            &nbsp;and our&nbsp;
```

Comment:
actually you want {' '} here not nbsp because you want these to be breakable on spaces

Reply:
I had used {' '} but yarn fix removed them later :/